distributed/tests/test_scheduler.py

```diff
@@ -624,7 +624,7 @@ def func(first, second):
         secede()
         second.wait()
 
-    fs = c.map(func, [first] * 5, [second] * 5)
+    fs = c.map(func, [first] * 5, [second] * 5, key=[f"x{i}" for i in range(5)])
```

Comment:
With https://github.com/dask/dask/pull/10883, func no longer generates tokenizes to 5 different random values, so we ended up with a single key.